docs/enterprise/active-learning/active_learning.md

```diff
@@ -191,11 +218,27 @@ the dataset representing the true production distribution, before any model is t
 
 Example client usage:
 ```python
+import cv2
 from inference_sdk import InferenceHTTPClient
 
+image = cv2.imread("<path_to_your_image>")
 LOCALHOST_CLIENT = InferenceHTTPClient(
     api_url="http://127.0.0.1:9001",
     api_key="XXX"
 )
 LOCALHOST_CLIENT.infer(image, model_id="asl-poly-instance-seg/0")
-```
\ No newline at end of file
+```
+
+## Parameters of requests to `inference` server influencing the Active Learning data collection
+There are few parameters that can be added to request to influence how data collection works, in particular:
+
+- `disable_active_learning` - to disable functionality at the level of single request (if for some reason you do not 
+want input data to be collected - useful for testing purposes)
+- `active_learning_target_dataset` - making inference from specific model (let's say `project_a/1`), when we want
+to save data in another project `project_b` - the latter should be pointed by this parameter. **Please remember that
+you cannot use different type of models in `project_a` and `project_b` - if that is the case - data will not be 
+registered)
+- `active_learning_api_key` - in rare cases (for instance cross-workspace data registration) one can provide auxiliary
```

Comment:
I'm not sure I understand when you would use this. For Universe the API key from your own workspace works to infer against public models.

Reply:
this makes the following action to be possible.
Someone has two workspaces - `A` and `B`. Workspace `A` has project `x` with model trained, They want to use model `x/1` and register data in project `y` from workspace `B`.
Not sure to what degree that feature would be useful for people - I do see this applicable for internal use-cases. I was asking about that couple of times at slack, without clear feedback received - so I implemented the thing as that was easy extension of changes introduced in this PR. We can obviously revert the think and that will be very quick.

Reply:
resolved at slack